
def check_fault():
    pass